azure-pipelines.yml

```diff
@@ -0,0 +1,39 @@
+trigger:
+- master
+
+pool:
+  vmImage: 'windows-latest'
+
+variables:
+  solution: '**/*.sln'
+  buildPlatform: 'Any CPU'
+  buildConfiguration: 'Release'
+  isTagBuild: $[startsWith(variables['Build.SourceBranch'], 'refs/tags/')]
+
+steps:
+- task: DotNetCoreCLI@2
+  displayName: restore
+  inputs:
+    command: 'restore'
+    feedsToUse: 'select'
+
+- task: DotNetCoreCLI@2
+  displayName: build solutions
+  inputs:
+    command: 'build'
+
+- task: DownloadSEcureFile@1
+  name: devSettings
+  displayName: fetch development settings
+  inputs:
+    secureFile: 'developer-settings.json'
+
+- task: PowerShell@2
```

Comment:
This is the problem, try https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-powershell?view=azure-devops instead.